src/CoreBundle/private/library/classes/TTemplateTools.class.php

```diff
@@ -9,9 +9,14 @@
  * file that was distributed with this source code.
  */
 
+/**
+ * @deprecated since 6.3.0 - view should use twig and twig filters instead.
+ */
 class TTemplateTools
 {
     /**
+     * @deprecated since 6.3.0 - use twig templates instead.
```

Comment:
We don't need to mark every method as deprecated - deprecating the class is enough.

Reply:
Are you sure that usages of a deprecated class are marked in IDEs? Not only if the method is deprecated?

Reply:
removed the method deprecations

Reply:
Calling `new` will be marked and using the class name, e.g. for static method calls, but not every method. Normally this is enough for developers to see they should use something else (and there's the UPGRADE guide and tool support in addition).